deploy/templates/nstemplatetiers/namespaces/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+advanced-code: "5c0e0f2"
```

Comment:
I guess this is a zombi from another PR?

Reply:
yes, let me remove these 2 `metadata.yaml` files. There was no reason to have them committed! 🤦‍♂

Reply:
removed in 9424627